src/sentry/sentry_metrics/indexer/postgres_v2.py

```diff
@@ -19,6 +20,10 @@
 # only used to compare to the older version of the PGIndexer
 _INDEXER_CACHE_FETCH_METRIC = "sentry_metrics.indexer.memcache.fetch"
 
+indexer_cache = StringIndexerCache(
+    **settings.SENTRY_STRING_INDEXER_CACHE_OPTIONS, partition_key=PartitionKey.POSTGRES
```

Comment:
We could leave this as is if we wanted and just do 

```python
StringIndexerCache(
    **settings.SENTRY_STRING_INDEXER_CACHE_OPTIONS, partition_key=PartitionKey.CLOUD_SPANNER
)
```

for the new implementation thats coming.